### DISCUSSION TITLE
Postman Collection

### DISCUSSION THREAD

**Discussion by thesilversverker** [hargata/lubelog#580] [General]
I made a basic postman collection for playing around & understanding the lubelogger API - should I just host that myself & link here, or is there a better way to provide that for others to use?

Nothing big, just saves some time for those of us who like pretty UI.

**Comment by hargata** [comment]
Official Postman Collection: https://github.com/hargata/lubelog_scripts/blob/main/misc/LubeLogger.postman_collection.json
